### ISSUE TITLE
Translation of Learning Path to Spanish

### ISSUE THREAD

**Issue by mberasategi** [github/training-kit#583]
## Overview
**TL;DR**
<One to two sentence description of the issue you are encountering or trying to solve.>
I'm thinking os translating the [Learning Path](https://services.github.com/on-demand/resources/learning-path/) into Spanish, but have some doubts on how to do it.

#### Reasoning

I decided to start working with Github with my Communication Degree students this fall semester. They are complete newbies and learning to become journalists, so not into the developer profile at all. It is going to require me preparing very basic materials to introduce version control, git, Github, Markdown, and even plain-text editing I'm afraid. So I just thought that if they could have this transalated _in situ_ instead of given by me through some of our school platforms, it will be more convenient for them, it could work as a demonstration on how to collaborate and of course anyone else reading Spanish could take advantage.

My questions are,

1. which are the files that need to be edited, and
1. what naming convention should I use for translated files?

Regarding the first question, I see the learning path index is in `https://github.com/github/training-kit/blob/master/resources/learning-path/index.html` but actual path files are in `https://github.com/github/training-kit/tree/master/paths`. Not sure if that's all or I am missing something. 

And about the second, I've seen the debate on naming conventions but can only find them in `downloads`, so not sure if that is the rule also for everything else.  Should I create `paths/es_ES` and have everything translated there? Or `paths/intro-to-github/es_ES` (and `resources/learning-path/es_ES` for the index)?

Also, I'm not sure about how this would be available on the interface for users, or if translations of the site are planned at all. So let me know!

**Comment by beardofedu** [comment]
@mberasategi First off, this is super exciting! 🎉  🎉  🎉  🎉  🎉  So the Training Path file that you want to modify is `training-kit/resources/learning-path/index.html`. For naming conventions, since our folder structure currently has the learning-path in its own folder,  I think the translated content should exist within that folder. So something like: `training-kit/resources/learning-path/es_index.html` should work. 

> I'm not sure about how this would be available on the interface for users

I think this is something that @github/services-training can start discussing and working out, but bottom line is, if we have the content it will be displayed some how :smile.

> or if translations of the site are planned at all

We currently don't have a project plan for translating the content for On Demand, however, we love it ❤️ when contributors like yourself jump in and help out.

**Comment by mberasategi** [comment]
Great to know! I'm going to start straight away then since classes begin this week :open_mouth: 

Correct me if I'm wrong, fork, translate, commit/push, pull request?

**Comment by mberasategi** [comment]
Done the first pull request (#586) with the translation of the first pages. I hope it's enough to test how the interface would work and to get everything started...

**Comment by mberasategi** [comment]
Not sure if we should keep discussion about the "how"s here or in the pull request discussion. Anyway, I've just found that part of the content is in `_includes` (and can't figure out the reasoning for some being directly in the path, some being outside. I guess it's reused somewhere). The thing is, path trouble again, I suggest

- append `-es` to filename of files in root, like `_includes/intro-video.html` to `_includes/intro-video-es.html`, and
- create `-es` dirs for directories, like `_includes/activity/intro-to-github/00-create-github-account.md` to `_includes/activity-es/intro-to-github/00-create-github-account.md`? Or `_includes/activity/intro-to-github-es/00-create-github-account.md`, since there are no files directly into `_includes/activity`?

Let me know. Thanks!

**Comment by hectorsector** [comment]
👋 @mberasategi I think you've already translated the Learning Path in #586. I believe all that remains is translating the title. What do you think?

**Comment by mberasategi** [comment]
Hi @hectorsector sorry for the delay. I think you mean the "On Demand Training" title, right?

I would go for something like "Formación a demanda"/ "Formación a medida" (more literal) or "Aprende a tu ritmo"/"Aprende a tu propio ritmo" or "Formación a tu (propio) ritmo" (literally "Learn/Training at your own pace").
